Sources/swift-format/Utilities/TTY.swift

```diff
@@ -11,6 +11,9 @@
 //===----------------------------------------------------------------------===//
 
 import Foundation
+#if canImport(Android)
```

Comment:
I tend to prefer `#if os(Android)` - this isn't about importing the module if it is available, this is about OS specific support.